Meadowlark-js/packages/meadowlark-authz-server/src/validation/BodyValidation.ts

```diff
@@ -34,6 +34,8 @@ function validateBody(body: object, schema: object, validateFunction: ValidateFu
         .map((property) => property.params.additionalProperty as string)
     : [];
 
+  // Will go through all additional properties found and tried to find match with the required
+  // properties. It is important to update the threshold for didYouMean to not allow words with similar lengths or typos.
```

Comment:
This is slightly helpful - but I could have figured this much out pretty quickly just reading the code. What future me will have trouble with: _why_ are we doing this?